LimeBean/Bean.cs

```diff
@@ -64,6 +64,13 @@ public partial class Bean {
             return this;
         }
 
+        /// <summary>
+        /// Retrieve the name of each Column held in this Bean
+        /// </summary>
+        public IEnumerable<string> Columns {
+            get { return ((Dictionary<string, object>)_props).Keys; } 
```

Comment:
You know what you're right. Habit picked up from wanting List<> functionality on IEnumerable<>